### ISSUE TITLE
Populate reserve_jobs = True not possible on Windows

### ISSUE THREAD

**Issue by horsto** [datajoint/datajoint-python#470]
Hi, I think I discovered a bug in datajoint python on Windows. 
I was trying to run ``.populate(reserve_jobs=True)`` on a table, but this ran into ``AttributeError: module 'os' has no attribute 'uname'``. This seems to be windows specific, since it works under ubuntu. 
Thanks!

**Comment by ixcat** [comment]
Hello -

Indeed os.uname is a unix specific function.

I made a patch as follows which *should* fix it but we should probably test it a bit on our side before merging. I don't have a 100% windows setup handy at the moment so couldn't test here just yet other than seeing that the files load in an interpreter and still seem to work on the unix side.

If you're using the pip installed version, the file should be somewhere under your `%LOCALAPPDATA%` area which is usually just `AppData` under your account files.

(here it was `%LOCALAPPDATA%\Programs\Python\python36\Lib\site-packages\datajoint\jobs.py`) - if you make the changes as outlined below it should hopefully work.

```
diff --git a/datajoint/jobs.py b/datajoint/jobs.py
index 50d53c9..295c6a6 100644
--- a/datajoint/jobs.py
+++ b/datajoint/jobs.py
@@ -1,6 +1,7 @@
 from _decimal import Decimal
 from .hash import key_hash
 import os
+import platform
 import pymysql
 from .base_relation import BaseRelation
 from . import DataJointError
@@ -78,7 +79,7 @@ class JobTable(BaseRelation):
             table_name=table_name,
             key_hash=key_hash(key),
             status='reserved',
-            host=os.uname().nodename,
+            host=platform.node(),
             pid=os.getpid(),
             connection_id=self.connection.connection_id,
             key=self.packable_or_none(key),
@@ -113,7 +114,7 @@ class JobTable(BaseRelation):
         self.insert1(
             dict(job_key,
                  status="error",
-                 host=os.uname().nodename,
+                 host=platform.node(),
                  pid=os.getpid(),
                  connection_id=self.connection.connection_id,
                  user=self._user,
```

**Comment by horsto** [comment]
Thanks! That worked.

**Comment by dimitri-yatsenko** [comment]
fixed with #471
